browser/ui/views/playlist/playlist_action_icon_view.cc

```diff
@@ -0,0 +1,117 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/views/playlist/playlist_action_icon_view.h"
+
+#include "brave/app/vector_icons/vector_icons.h"
+#include "brave/browser/ui/views/playlist/playlist_action_bubble_view.h"
+#include "ui/base/metadata/metadata_impl_macros.h"
+
+PlaylistActionIconView::PlaylistActionIconView(
+    CommandUpdater* command_updater,
+    Browser* browser,
+    IconLabelBubbleView::Delegate* icon_label_bubble_delegate,
+    PageActionIconView::Delegate* page_action_icon_delegate)
+    : PageActionIconView(command_updater,
+                         0,
```

Comment:
Optional & not super important (and if you decide to do it, it doesn't need to be this PR):
If we add a command for this it'll show up in our shortcuts page & in the command palette.